LayoutTests/platform/ipad/TestExpectations

```diff
@@ -127,8 +127,8 @@ media/mediacapabilities/mediacapabilities-allowed-containers.html [ Pass ]
 
 # webkit.org/b/269393 ([ iOS wk2 ] 3 iOS orientation(layout-tests) are constant text failures)
 compositing/backing/page-scale-overlap-in-iframe.html [ Failure ]
-fast/events/ios/rotation/zz-no-rotation.html [ Failure ]
-fast/screen-orientation/natural-orientation.html [ Failure ]
+fast/events/ios/rotation/zz-no-rotation.html [ Failure Crash ]
```

Comment:
@rniwa, um, I'm unsure what I'm doing wrong here... I was expecting:
LayoutTests/platform/ipad/fast/events/ios/rotation/zz-no-rotation-expected.txt

To pass... but when I run `run-webkit-tests --reset-results --ipad-simulator` it seems to reset the "ios" results.